### PR TITLE
Add breadcrumbs to docs

### PR DESCRIPTION
**Pull request by corwintines**
- Adds Breadcrumb component for docs pages

### CODE REVIEW

**Review by wackerow — APPROVED** [review]

File: src/components/docs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+export const Breadcrumbs: FC<Props> = ({ router }) => {
+  let pathSplit = router.asPath.split('/')
+  pathSplit = pathSplit.splice(1,pathSplit.length)
+
+  return (
+    <Breadcrumb mb={10}>
+      {pathSplit.map((path: string, idx: number) => {
+        return (
+          <BreadcrumbItem key={path}>
+            <BreadcrumbLink
```
Comment (nhsz) [inline src/components/docs/Breadcrumbs.tsx:17]: should be wrapped inside a `next/link` component to avoid full page reload

**Review by nhsz — CHANGES_REQUESTED** [review]
Great! Just a small suggestion to avoid page reloads on navigation

File: src/components/docs/Breadcrumbs.tsx (hunk 1 above)
Comment (corwintines) [reply src/components/docs/Breadcrumbs.tsx:17]: Sounds good, will try out.

File: src/components/docs/Breadcrumbs.tsx [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
       {pathSplit.map((path: string, idx: number) => {
         return (
           <BreadcrumbItem key={path}>
-            <BreadcrumbLink
-              color={ idx+1 === pathSplit.length ? 'body' : 'primary' }
-              href={`/${pathSplit.slice(0,idx+1).join('/')}`}
-            >
-              {path}
-            </BreadcrumbLink>
+            <NextLink href={`/${pathSplit.slice(0, idx + 1).join('/')}`} passHref>
```
Comment (nhsz) [inline src/components/docs/Breadcrumbs.tsx:18]: I meant to wrap `BreadcrumbLink` (`BreadcrumbLink`is more semantically correct here vs  Chakra's`Link`), does this work?

```
<NextLink href={`/${pathSplit.slice(0, idx + 1).join('/')}`} passHref>
  <BreadcrumbLink color={ idx+1 === pathSplit.length ? 'body' : 'primary' }>
    {path}
  </BreadcrumbLink>
</NextLink>
```

File: src/components/docs/Breadcrumbs.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  let pathSplit = router.asPath.split('/');
+  pathSplit = pathSplit.splice(1, pathSplit.length);
+
+  return (
+    <Breadcrumb mb={10}>
+      {pathSplit.map((path: string, idx: number) => {
+        return (
+          <BreadcrumbItem key={path}>
+            <NextLink href={`/${pathSplit.slice(0, idx + 1).join('/')}`} passHref>
+              <Link color={idx + 1 === pathSplit.length ? 'body' : 'primary'}>{path}</Link>
```
Comment (wackerow) [inline src/components/docs/Breadcrumbs.tsx:19]:
```suggestion
              <BreadcrumbLink color={ idx+1 === pathSplit.length ? 'body' : 'primary' }>
                {path}
              </BreadcrumbLink>
```

File: src/components/docs/Breadcrumbs.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { Breadcrumb, BreadcrumbItem, Link } from '@chakra-ui/react';
```
Comment (wackerow) [inline src/components/docs/Breadcrumbs.tsx:1]:
```suggestion
import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from '@chakra-ui/react';
```

**Review by wackerow — COMMENTED** [review]
What type of casing are we going with here?

This PR is all lowercase:
image

versus the Figma:
image

Which seems to be title case, except the last item makes it unclear as that is lowercase.

File: src/components/docs/Breadcrumbs.tsx (hunk 2 above)
Comment (corwintines) [reply src/components/docs/Breadcrumbs.tsx:18]: Ah good point

**Comment by corwintines** [comment]
> What type of casing are we going with here?
> 
> This PR is all lowercase: image
> 
> versus the Figma: image
> 
> Which seems to be title case, except the last item makes it unclear as that is lowercase.

I'll bring it up with @nloureiro tomorrow morning. I think we could merge it either way here and fix that

**Review by nhsz — APPROVED** [review]

**Pull request merged**